.browserslistrc

```diff
@@ -0,0 +1 @@
+defaults
```

Comment:
What is that for?

Reply:
We don't know.

Reply:
Should be list of browsers which we use. We'll hardly need it, so deleted